WORKSPACE.bazel

```diff
@@ -73,6 +75,7 @@ llvm_toolchain(
             "-stdlib=libc++",
```

Comment:
Does this still work? I think the grailbio toolchain docs say specifying sysroot results in using libstdc++:

https://github.com/grailbio/bazel-toolchain/blob/master/README.md#cross-compilation

Reply:
We can take a look at it later. I haven't paid too much attention to which standard library is getting linked in. I also don't think the provided sysroot exposes a C++ standard library.

Reply:
```
 "external/llvm_16_0_toolchain_llvm/bin/ld.lld" --sysroot=external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot -pie -z relro --hash-style=gnu --eh-frame-hdr -m elf_x86_64 -dynamic-linker /lib64/ld-linux-x86-64.so.2 -o bazel-out/k8-fastbuild/bin/huffman/test/bit_test external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/Scrt1.o external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/crti.o external/llvm_16_0_toolchain_llvm/lib/clang/16/lib/x86_64-unknown-linux-gnu/clang_rt.crtbegin.o -Lexternal/llvm_16_0_toolchain_llvm/bin/../lib/x86_64-unknown-linux-gnu -Lexternal/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/lib/../lib64 -Lexternal/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64 -Lexternal/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/lib -Lexternal/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib -S -lm --build-id=md5 --hash-style=gnu -z relro -z now -l:libc++.a -l:libc++abi.a -l:libunwind.a -lpthread -ldl bazel-out/k8-fastbuild/bin/huffman/test/_objs/bit_test/bit_test.pic.o --verbose --rpath=/home/oliver/.cache/bazel/_bazel_oliver/adb62031b64cc76568fe096c0cc5a5f9/external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/lib --rpath=/home/oliver/.cache/bazel/_bazel_oliver/adb62031b64cc76568fe096c0cc5a5f9/external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib --rpath=/home/oliver/.cache/bazel/_bazel_oliver/adb62031b64cc76568fe096c0cc5a5f9/external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/../lib64 --dynamic-linker=/home/oliver/.cache/bazel/_bazel_oliver/adb62031b64cc76568fe096c0cc5a5f9/external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/lib/ld-linux-x86-64.so.2 -lstdc++ -lm external/llvm_16_0_toolchain_llvm/lib/clang/16/lib/x86_64-unknown-linux-gnu/libclang_rt.builtins.a -lc external/llvm_16_0_toolchain_llvm/lib/clang/16/lib/x86_64-unknown-linux-gnu/libclang_rt.builtins.a external/llvm_16_0_toolchain_llvm/lib/clang/16/lib/x86_64-unknown-linux-gnu/clang_rt.crtend.o external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/crtn.o
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/Scrt1.o
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/crti.o
ld.lld: external/llvm_16_0_toolchain_llvm/lib/clang/16/lib/x86_64-unknown-linux-gnu/clang_rt.crtbegin.o
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/libm.so
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/lib64/libm.so.6
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/lib64/libmvec.so.1
ld.lld: external/llvm_16_0_toolchain_llvm/bin/../lib/x86_64-unknown-linux-gnu/libc++.a
ld.lld: external/llvm_16_0_toolchain_llvm/bin/../lib/x86_64-unknown-linux-gnu/libc++abi.a
ld.lld: external/llvm_16_0_toolchain_llvm/bin/../lib/x86_64-unknown-linux-gnu/libunwind.a
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/libpthread.a
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/libdl.a
ld.lld: bazel-out/k8-fastbuild/bin/huffman/test/_objs/bit_test/bit_test.pic.o
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/libstdc++.so
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/libm.so
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/lib64/libm.so.6
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/lib64/libmvec.so.1
ld.lld: external/llvm_16_0_toolchain_llvm/lib/clang/16/lib/x86_64-unknown-linux-gnu/libclang_rt.builtins.a
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/libc.so
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/lib64/libc.so.6
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib64/libc_nonshared.a
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/lib64/ld-linux-x86-64.so.2
ld.lld: external/llvm_16_0_toolchain_llvm/lib/clang/16/lib/x86_64-unknown-linux-gnu/libclang_rt.builtins.a
ld.lld: external/llvm_16_0_toolchain_llvm/lib/clang/16/lib/x86_64-unknown-linux-gnu/clang_rt.crtend.o
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/crtn.o
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/libpthread.a
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/libdl.a
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/libpthread.a
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/libpthread.a
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/libpthread.a
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/libpthread.a
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/libpthread.a
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/libpthread.a
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/libpthread.a
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/librt.a
Target //huffman/test:bit_test up-to-date:
  bazel-bin/huffman/test/bit_test
```

Reply:
looks okay to me now:
```
 "external/llvm_16_0_toolchain_llvm/bin/ld.lld" --sysroot=external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot -pie -z relro --hash-style=gnu --eh-frame-hdr -m elf_x86_64 -dynamic-linker /lib64/ld-linux-x86-64.so.2 -o bazel-out/k8-fastbuild/bin/huffman/test/bit_test external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/Scrt1.o external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/crti.o external/llvm_16_0_toolchain_llvm/lib/clang/16/lib/x86_64-unknown-linux-gnu/clang_rt.crtbegin.o -Lexternal/llvm_16_0_toolchain_llvm/bin/../lib/x86_64-unknown-linux-gnu -Lexternal/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/lib/../lib64 -Lexternal/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64 -Lexternal/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/lib -Lexternal/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib -S -lm --build-id=md5 --hash-style=gnu -z relro -z now -l:libc++.a -l:libc++abi.a -l:libunwind.a -lpthread -ldl bazel-out/k8-fastbuild/bin/huffman/test/_objs/bit_test/bit_test.pic.o --verbose --rpath=/home/oliver/.cache/bazel/_bazel_oliver/adb62031b64cc76568fe096c0cc5a5f9/external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/lib --rpath=/home/oliver/.cache/bazel/_bazel_oliver/adb62031b64cc76568fe096c0cc5a5f9/external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib --rpath=/home/oliver/.cache/bazel/_bazel_oliver/adb62031b64cc76568fe096c0cc5a5f9/external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/../lib64 --dynamic-linker=/home/oliver/.cache/bazel/_bazel_oliver/adb62031b64cc76568fe096c0cc5a5f9/external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/lib/ld-linux-x86-64.so.2 -lc++ -lm external/llvm_16_0_toolchain_llvm/lib/clang/16/lib/x86_64-unknown-linux-gnu/libclang_rt.builtins.a -lc external/llvm_16_0_toolchain_llvm/lib/clang/16/lib/x86_64-unknown-linux-gnu/libclang_rt.builtins.a external/llvm_16_0_toolchain_llvm/lib/clang/16/lib/x86_64-unknown-linux-gnu/clang_rt.crtend.o external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/crtn.o
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/Scrt1.o
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/crti.o
ld.lld: external/llvm_16_0_toolchain_llvm/lib/clang/16/lib/x86_64-unknown-linux-gnu/clang_rt.crtbegin.o
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/libm.so
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/lib64/libm.so.6
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/lib64/libmvec.so.1
ld.lld: external/llvm_16_0_toolchain_llvm/bin/../lib/x86_64-unknown-linux-gnu/libc++.a
ld.lld: external/llvm_16_0_toolchain_llvm/bin/../lib/x86_64-unknown-linux-gnu/libc++abi.a
ld.lld: external/llvm_16_0_toolchain_llvm/bin/../lib/x86_64-unknown-linux-gnu/libunwind.a
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/libpthread.a
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/libdl.a
ld.lld: bazel-out/k8-fastbuild/bin/huffman/test/_objs/bit_test/bit_test.pic.o
ld.lld: external/llvm_16_0_toolchain_llvm/bin/../lib/x86_64-unknown-linux-gnu/libc++.a
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/libm.so
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/lib64/libm.so.6
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/lib64/libmvec.so.1
ld.lld: external/llvm_16_0_toolchain_llvm/lib/clang/16/lib/x86_64-unknown-linux-gnu/libclang_rt.builtins.a
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/libc.so
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/lib64/libc.so.6
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib64/libc_nonshared.a
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/lib64/ld-linux-x86-64.so.2
ld.lld: external/llvm_16_0_toolchain_llvm/lib/clang/16/lib/x86_64-unknown-linux-gnu/libclang_rt.builtins.a
ld.lld: external/llvm_16_0_toolchain_llvm/lib/clang/16/lib/x86_64-unknown-linux-gnu/clang_rt.crtend.o
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/crtn.o
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/libpthread.a
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/libdl.a
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/libpthread.a
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/libpthread.a
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/libpthread.a
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/libpthread.a
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/libpthread.a
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/libpthread.a
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/libpthread.a
ld.lld: external/gcc_12_2_toolchain_files/x86_64-buildroot-linux-gnu/sysroot/usr/lib/../lib64/librt.a
Target //huffman/test:bit_test up-to-date:
  bazel-bin/huffman/test/bit_test
```